### PR TITLE
reports table is created.

### PR DESCRIPTION
**Pull request by Ayush8923**
Targets #1234 

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/SalesAutomation/Database/Migrations/2021_12_27_175619_reports_table.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class ReportsTable extends Migration
```
Comment (GauravGusain98) [inline Modules/SalesAutomation/Database/Migrations/2021_12_27_175619_reports_table.php:7]: The the file name and class name should be create_reports_table

File: Modules/SalesAutomation/Database/Migrations/2021_12_27_175619_reports_table.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('reports', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->text('description');
```
Comment (GauravGusain98) [inline Modules/SalesAutomation/Database/Migrations/2021_12_27_175619_reports_table.php:19]: This is a nullable field

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments.

**Comment by Ayush8923** [comment]
I am creating another pull request instead. That's why I am closing this PR.

**Pull request closed without merging**